.gitignore

```diff
@@ -43,3 +43,7 @@ dist/
 # !assets/css/prism.css
 # !assets/css/styleguide.scss
 # !assets/js/styleguide.js
+
+#Ignore IntelliJ IDE .idea file
+
+.idea/
```

Comment:
@anuragmundada-reisys - Thanks for the PR! Is it possible to remove this line from the `.gitignore`, I'm not sure if anyone else is using IntelliJ. If you are using GitHub Desktop - or something similar - if the `.idea` file/folder pops up you should be able to do a right/option-click and discard changes on that file.

"Ignoring" - actually modifies the `.gitignore` - I just found that out yesterday myself. :)